lib/deps.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const kModuleError = Symbol('Error Loading Module');
```

Comment:
This will look pretty verbose if you ever print it out:
```
> class Test { constructor() { this[Symbol('Module Loading Error')] = true; } }
undefined
> x = new Test()
```
suggest just `const kModuleError = Symbol('moduleError')`